### PR TITLE
Fix page being hardcoded to 1.

### LINKED ISSUE #3: Page is hardcoded to 1
It doesn't come back with the correct page number when you pass in something to the URL. ex:
`?page=4` always comes back with `{...,"page":1,...}`

### PR DESCRIPTION
**Pull request by bbean86**
Fixes #3

### CODE REVIEW

File: lib/sapine.rb [hunk 1]
```diff
@@ -78,6 +79,10 @@ def per_page
     params[:per_page] ? params[:per_page].to_i : DEFAULT_PER_PAGE
   end
 
+  def page
+    params[:page] ? params[:page].to_i : DEFAULT_PAGE
```
Comment (teubanks) [inline lib/sapine.rb:83]: What happens if I request page 3000 and there's only 10 pages?

Comment (teubanks) [inline lib/sapine.rb:83]: Maybe there should be a `[params[:page], total_pages].min` ?

Comment (jspies) [inline lib/sapine.rb:83]: ActiveRecord should just return empty results. If that's ok with you guys, you can leave as is.

**Comment by jspies** [comment]
:ship: I'm impressed it took this long to find. Guess we weren't using the meta data much.

**Comment by bbean86** [comment]
I'm good with it. @jspies you'll have to merge since it's your repo

**Comment by teubanks** [comment]
:shipit: :beastmode:

**Pull request merged**